pkg/bucketeer/sdk.go

```diff
@@ -347,6 +347,22 @@ func getEvaluationDetails[T model.EvaluationValue](
 ) model.BKTEvaluationDetails[T] {
 	var err error
 	var value T
+
+	err = validateGetEvaluationRequest(user, featureID)
```

Comment:
We also need the validation here.
https://github.com/bucketeer-io/go-server-sdk/blob/afc407461536bfe795d8d7c952c7970ec4f114ae/pkg/bucketeer/sdk.go#L467-L471

Reply:
I have added the validation